raiden/blockchain_events_handler.py

```diff
@@ -256,6 +271,17 @@ def handle_channel_withdraw(raiden: "RaidenService", event: Event):
         )
         msg = "Failed to find channel state after withdraw"
         assert channel_state is not None, msg
+
+        update_fee_statechange = ActionChannelUpdateFee(
+            canonical_identifier=previous_channel_state.canonical_identifier,
+            flat_fee=previous_channel_state.fee_schedule.flat,
+            proportional_fee=previous_channel_state.fee_schedule.proportional,
+            use_imbalance_penalty=previous_channel_state.fee_schedule.imbalance_penalty
+            is not None,
+        )
+        raiden.handle_and_track_state_change(update_fee_statechange)
+
+        # Update PFS about changed fees for this channel, when not in private mode
         update_path_finding_service_from_channel_state(raiden=raiden, channel_state=channel_state)
```

Comment:
Not sure here and in similar places:
As I emit a second state change the channel state is changed. However, I don't get the updated chain state and channel state from the `raiden`  instance. My guess is that I need to do this, otherwise the update will be lost until the next update.
Is that right, @hackaugusto ?